emulators/qemu/Portfile

```diff
@@ -114,6 +119,24 @@ pre-configure {
     configure.args-append --extra-ldflags="${configure.ldflags}"
 }
 
+# out of tree hvf support for Apple Silicon
+# https://patchwork.kernel.org/project/qemu-devel/list/?series=418581
+patchfiles-append       patch-hvf-Add-hypervisor-entitlement-to-output-binaries.diff \
+                        patch-hvf-x86-Remove-unused-definitions.diff \
+                        patch-hvf-Move-common-code-out.diff \
+                        patch-hvf-Introduce-hvf-vcpu-struct.diff \
+                        patch-arm-Set-PSCI-to-0.2-for-HVF.diff \
+                        patch-hvf-Simplify-post-reset-init-loadvm-hooks.diff \
+                        patch-hvf-Add-Apple-Silicon-support.diff \
+                        patch-arm-Add-Hypervisor.framework-build-target.diff \
+                        patch-arm-hvf-Add-a-WFI-handler.diff \
+                        patch-hvf-arm-Add-support-for-GICv3.diff \
+                        patch-hvf-arm-Implement--cpu-host.diff
+
+post-configure {
+    system "chmod +x ${worksrcpath}/scripts/entitlement.sh"
```

Comment:
No need to spawn a shell for this, this can be done from Tcl directly:
`file attributes ${worksrcpath}/scripts/entitlement.sh -permissions +x`